code/game/objects/items/toys.dm

```diff
@@ -217,8 +217,8 @@
  */
 
 /obj/item/toy/crossbow
-	name = "foam dart crossbow"
-	desc = "A weapon favored by many overactive children. Ages 8 and up."
+	name = "mini energy-crossbow"

```

Comment:
N O

Reply:
why are you fine with everything else being renamed but not the energy crosbow

Reply:
I'm not a fan of them being renamed either, this is just the one I take the most offense to.